plaso/parsers/sqlite.py

```diff
@@ -413,7 +413,7 @@ def ParseFileEntry(self, parser_mediator, file_entry):
       file_entry (dfvfs.FileEntry): file entry to be parsed.
 
     Raises:
-      UnableToParseFile: when the file cannot be parsed.
+      WrongParser: when the file cannot be parsed.
```

Comment:
does not appear to be raised in this method, remove from docstring?

Reply:
Done